recipes/libexif/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import contextlib
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LibexifConan(ConanFile):
+    name = "libexif"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libexif.github.io/"
+    license = "LGPL-2.1"
+    description = "libexif is a library for parsing, editing, and saving EXIF data."
+    topics = ("exif", "metadata", "parse", "edit")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    @contextlib.contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self):
+                env = {
+                    "CC": "cl -nologo",
```

Comment:
Try this too when the build fails.
```suggestion
                    "AR": "lib",
                    "CC": "cl -nologo",
```